cinema/views.py

```diff
@@ -0,0 +1,38 @@
+from django.http import JsonResponse
```

Comment:
The import of `JsonResponse` is unnecessary since you are using `Response` from Django REST Framework for returning responses. Consider removing this import to clean up the code.